### PR TITLE
Add fugu 1.1.1

### PR DESCRIPTION
**Pull request by neronplex**
Docker command wrapper tool.

### CODE REVIEW

**Comment by dunn** [comment]
Did you mean to open this in homebrew-binary (#41869)?

**Comment by neronplex** [comment]
"Homebrew-binary" is later to know was that it is a policy that does not accept as a principle that can be compiled in the open source.Because it was confirmed that can be compiled in the "homebrew", it issued a request this time again.

**Comment by dunn** [comment]
Then it will have to be actually built from source; right now the formula just installs the precompiled binary.

**Comment by neronplex** [comment]
Whether you should have specified the compressed file in the source code in the Formula, but this has not been compiled with homebrew indeed?

**Comment by dunn** [comment]
Yes, "built from source", means it is compiled during the installation by Homebrew, and not previously by the upstream developer.

**Comment by dunn** [comment]
See [the platinum searcher](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/the_platinum_searcher.rb) for an example of a Go project that's built from source.

**Comment by neronplex** [comment]
Oh, I see. Please wait...

**Comment by neronplex** [comment]
Fixed formula. Please review it.

File: Library/Formula/fugu.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  go_resource "github.com/howeyc/gopass" do
+    url "https://github.com/howeyc/gopass.git", :revision => "10b54de414cc9693221d5ff2ae14fd2fbf1b0ac1"
+  end
+
+  go_resource "golang.org/x/crypto" do
+    url "https://github.com/golang/crypto.git", :revision => "7f27901cb9636a61b9b0b1bab2679b2a9a12eb38"
+  end
+
+  go_resource "github.com/docker/docker" do
```
Comment (dunn) [inline Library/Formula/fugu.rb]: You already have `depends_on "docker"` above. Is it possible to avoid duplicating the dependency here?

**Comment by dunn** [comment]
The godeps file lists a lot more dependencies: https://github.com/mattes/fugu/blob/v1/Godeps/Godeps.json 

How are you getting by without them all?

**Comment by neronplex** [comment]
Some If you try to use the godep it becomes error in the package of the resolution. The package I was writing all by trial and error.

**Comment by neronplex** [comment]
It was modified to use the godep.

**Comment by dunn** [comment]
The test bot is failing to install:
```
==> Downloading https://github.com/mattes/fugu/archive/v1.1.1.tar.gz
Already downloaded: /Library/Caches/Homebrew/fugu-1.1.1.tar.gz
==> Verifying fugu-1.1.1.tar.gz checksum
tar xf /Library/Caches/Homebrew/fugu-1.1.1.tar.gz
Error: No such file or directory - /private/tmp/fugu20150804-51418-yxpg8a/fugu-1.1.1/src/github.com/tools/godep
```

**Comment by neronplex** [comment]
I'm sorry. Verification at whether hand was left cache was insufficient. In the most recent commit to being successful.

**Comment by neronplex** [comment]
ping.

**Comment by tdsmith** [comment]
Thanks for your pull request! :tada: Homebrew thrives because of contributions like this one. Pulled with small tweaks in b4d0783cca7dbd2.

**Comment by neronplex** [comment]
Thank you for approval!

**Pull request closed without merging**